src/main/java/org/swisspush/reststorage/EventBusAdapter.java

```diff
@@ -506,10 +590,40 @@ public HttpConnection connection() {
             return null;
         }
 
+        @Override
+        public HttpServerRequest streamPriorityHandler(Handler<StreamPriority> handler) {
+            return null;
+        }
+
+        @Override
+        public DecoderResult decoderResult() {
+            return null;
+        }
+
+        @Override
+        public Cookie getCookie(String name) {
+            return null;

```

Comment:
We should `throw new UnsupportedOperationException();` for those methods instead of returning `null`